pages/dkp/kaptain/2.0.0/install/air-gapped-2.2/index.md

```diff
@@ -0,0 +1,96 @@
+---
+layout: layout.pug
+navigationTitle: Add Kaptain air-gapped for DKP 2.2
+title: Add Kaptain to your DKP Catalog applications on air-gapped environments for DKP 2.2
+menuWeight: 12
+excerpt: Add Kaptain to your DKP Catalog applications on air-gapped environments for DKP 2.2
+beta: false
+enterprise: false
+---
+
+<p class="message--warning"><strong>WARNING: </strong>
+You can deploy Kaptain to a cluster in a selected workspace. If you do not intend to deploy Kaptain to a certain cluster, you must switch the workspace you are deploying to or move that cluster to another workspace.
+</p>
+
+## DKP 2.2 air-gapped installation
+
+Refer to [DKP install instructions][dkp_install], if you want to deploy Kaptain in a networked environment or to [DKP 2.2 air-gapped instructions][2.1_air] if you are deploying in DKP 2.1.
+
+Kaptain supports installation on an air-gapped (a.k.a. offline or private) DKP managed cluster. Before installing Kaptain, please follow the [air-gapped installation guide][konvoy-air-gap] to set up the air-gapped DKP managed cluster. The cluster admin is responsible for configuring the DKP cluster correctly and ensuring container images have been pre-loaded to the private registry before installing Kaptain.
+
+## Prerequisites
+
+-   A DKP cluster with the following Platform applications enabled:
+
+    - Istio
+    - Knative (optional, if KServe is configured to work in `RawDeployment` mode)
+
+-   [`kubectl`][kubectl] on your installation machine
+
+-   For customers deploying in a multi-cluster environment (Enterprise): Ensure you have configured [Kaptain to authenticate with a Management Cluster][dex].
+
+-   Ensure the following applications are enabled in Kommander. 
+
+    Review the [Kommander installation documentation][kommander-install] for more information.
+
+<!-- TODO: Clarify which DKP doc to reference here. -->
```

Comment:
Is this the document you were referring to? https://docs.d2iq.com/dkp/kommander/2.2/install/configuration/
Something like this: 
```suggestion
[Configure a Kommander installation](../../../../kommander/2.2/install/configuration) to include `dex`, `dex-k8s-authenticator`, `kube-prometheus-stack`, `istio`, `minio-operator`, `traefik`, `nvidia` and `knative` (Knative is optional if KServe is configured to work in `RawDeployment` mode).
```
Again, making wild guesses.